aws/resource_aws_iam_group_membership.go

```diff
@@ -35,6 +36,18 @@ func resourceAwsIamGroupMembership() *schema.Resource {
 				Required: true,
 				ForceNew: true,
 			},
+
+			"remove_unmanaged_members": &schema.Schema{
```

Comment:
Minor nitpick: As of Go 1.7+, the `&schema.Schema` after the attribute name declarations are unnecessary